website/content/en/preview/reference/metrics.md

```diff
@@ -10,254 +10,335 @@ description: >
 Karpenter makes several metrics available in Prometheus format to allow monitoring cluster provisioning status. These metrics are available by default at `karpenter.karpenter.svc.cluster.local:8000/metrics` configurable via the `METRICS_PORT` environment variable documented [here](../settings)
 ### `karpenter_build_info`
 A metric with a constant '1' value labeled by version from which karpenter was built.
+- Stability Level: STABLE
```

Comment:
It would be cool to eventually have the `Type` and `Labels` next to these similar to the upstream docs